crates/uv-fs/src/lib.rs

```diff
@@ -45,40 +46,34 @@ pub async fn read_to_string_transcode(path: impl AsRef<Path>) -> std::io::Result
 
 /// Create a symlink at `dst` pointing to `src`, replacing any existing symlink.
 ///
-/// On Windows, this uses the `junction` crate to create a junction point. The
-/// operation is _not_ atomic, as we first delete the junction, then create a
-/// junction at the same path.
-///
-/// Note that because junctions are used, the source must be a directory.
+/// On Windows, we emulate symlinks by writing a file with the target path.
 #[cfg(windows)]
 pub fn replace_symlink(src: impl AsRef<Path>, dst: impl AsRef<Path>) -> std::io::Result<()> {
-    // If the source is a file, we can't create a junction
-    if src.as_ref().is_file() {
-        return Err(std::io::Error::new(
-            std::io::ErrorKind::InvalidInput,
-            format!(
-                "Cannot create a junction for {}: is not a directory",
-                src.as_ref().display()
-            ),
-        ));
-    }
+    // First, attempt to create a file at the location, but fail if it doesn't exist.
```

Comment:
Does this intend to "fail if it _does_ exist"?

Reply:
Yes!